app/src/main/res/values/strings.xml

```diff
@@ -552,7 +552,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="exif_tag_name_software">Software</string>
 
   <string name="share_text">Upload photos to Wikimedia Commons directly from your phone. Download the Commons App now: %1$s</string>
-  <string name="share_via">Share app via...</string>
+  <string name="share_via">Share app via…</string>
```

Comment:
Using the elipsis character is recommended by lint but it is not a part of this ticket, probably something we have to discuss with Translatewiki

Reply:
That was a mistake on my end! I was trying to search for what a string Id meant in the xml file, and realized that I had removed something, so I ctrl+z. I didn’t think git would reflect it since I did not refactor anything.

Reply:
Well, we see the difference just because Git has detected it. So, it would be nice if you could revert it. You could use the following suggestion, if you wish:

```suggestion
  <string name="share_via">Share app via...</string>
```

FYI: Translatewiki handles the ellipisis character just fine. Here's an example of a string from another app which contains the ellipsis character and is translated appropriately: https://translatewiki.net/wiki/Wikimedia:Wikipedia-android-strings-login_in_progress_dialog_message/en

Reply:
Well, we see the difference just because Git has detected it. So, it would be nice if you could revert it. You could use the following suggestion, if you wish:

```suggestion
  <string name="share_via">Share app via...</string>
```

FYI: Translatewiki handles the ellipisis character just fine. Here's an example of a string from another app which contains the ellipsis character and is translated appropriately: https://translatewiki.net/wiki/Wikimedia:Wikipedia-android-strings-login_in_progress_dialog_message/en